### PR TITLE
Fixed:  (BHD) Treat Limited Freeleech as Freeleech

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
NO

#### Description
- Treat limited freeleech as freeleech
- finish todo now that the API is supposedly fixed
- correct the comment for min seed time

#### Screenshot (if UI related)
image

#### Todos
- [X] Test to see if it works - seems to work (search marked as FL | grabed w/o issue)

#### Issues Fixed or Closed by this PR

* Fixes: #268
* Closes: #269

### CODE REVIEW

File: src/NzbDrone.Core/Indexers/Definitions/BeyondHD.cs [hunk 1]
```diff
@@ -190,11 +190,11 @@ public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
                 throw new IndexerException(indexerResponse, $"Unexpected response status {indexerResponse.HttpResponse.StatusCode} code from API request");
             }
 
-            // TODO Have BHD fix their API response content type so we can proper check here
-            // if (!indexerResponse.HttpResponse.Headers.ContentType.Contains(HttpAccept.Json.Value))
-            // {
-            //     throw new IndexerException(indexerResponse, $"Unexpected response header {indexerResponse.HttpResponse.Headers.ContentType} from API request, expected {HttpAccept.Json.Value}");
-            // }
+            if (!indexerResponse.HttpResponse.Headers.ContentType.Contains(HttpAccept.Json.Value))
```
Comment (RobinDadswell) [inline src/NzbDrone.Core/Indexers/Definitions/BeyondHD.cs:193]: For Posterity: Forum post confirming this is now fixed so this is okay to be included

File: src/NzbDrone.Core/Indexers/Definitions/BeyondHD.cs [hunk 2]
```diff
@@ -217,10 +217,10 @@ public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
                     ImdbId = ParseUtil.GetImdbID(row.ImdbId).GetValueOrDefault(),
                     TmdbId = row.TmdbId.IsNullOrWhiteSpace() ? 0 : ParseUtil.CoerceInt(row.TmdbId.Split("/")[1]),
                     Peers = row.Leechers + row.Seeders,
-                    DownloadVolumeFactor = row.Freeleech ? 0 : row.Promo75 ? 0.25 : row.Promo50 ? 0.5 : row.Promo25 ? 0.75 : 1,
+                    DownloadVolumeFactor = row.Freeleech ? 0 : row.Promo75 ? 0.25 : row.Promo50 ? 0.5 : row.Promo25 ? 0.75 : row.Limited ? 0 : 1,
```
Comment (RobinDadswell) [inline src/NzbDrone.Core/Indexers/Definitions/BeyondHD.cs:220]: nit: do `row.Freelech || row.Limited ? 0 : ...`

**Review by RobinDadswell — CHANGES_REQUESTED** [review]
Small Nit, otherwise I think we are okay

**Comment by Qstick** [comment]
As another note, with our new flag system in prowlarr we can define indexer specific flags. This would allow us to call these out as limited-freeleech to give the user more information if we think it's worthwhile

**Review by Qstick — APPROVED** [review]

**Pull request merged**